virtme/guest/virtme-sshd-script

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Initialize ssh server for remote connections (option `--server ssh`)
+
+# Generate authorized_keys adding the host's user public keys.
+HOME=$(getent passwd "${ssh_user}" | cut -d: -f6)
+cat ${HOME}/.ssh/id_*.pub >> ${HOME}/.ssh/authorized_keys
+chown ${ssh_user} ${HOME}/.ssh/authorized_keys
+
+# Try to use the virtme-ng host keys if available, otherwise generate a new
+# key on the fly.
+HOST_KEY=${HOME}/.cache/virtme-ng/.ssh/id_rsa
+if [ ! -e "${HOST_KEY}" ]; then
+    ssh-keygen -t rsa -b 2048 -f "${HOST_KEY}" -N "" -q
```

Comment:
`mkdir -p` will be needed here.

But as mentioned above, I think it would be safer to do this here instead:

```
CACHE_DIR="{HOME}/.cache/virtme-ng/ssh"  ## or /virtme/?
mkdir -p "${CACHE_DIR}/etc/ssh"
ssh-keygen -A -f "${CACHE_DIR}" ## will not do anything if the keys exist
ARGS=(); for i in "${CACHE_DIR}"/etc/ssh/ssh_host_*_key; do ARGS+=(-h ${i}); done

(...)

/usr/sbin/sshd "${ARGS[@]}"

Reply:
About the cache dir, we already have `~/.cache/virtme-ng` (used to cache the precompiled mainline kernels), so we should use the same location.

Reply:
> About the cache dir, we already have `~/.cache/virtme-ng` (used to cache the precompiled mainline kernels), so we should use the same location.

Yes, I was not sure if we wanted a different cache for `virtme-run` and `virtme-ng`, to avoid any conflicts.

But fine to keep the same location.